test/functional/test_framework/wallet.py

```diff
@@ -338,6 +337,19 @@ def sendrawtransaction(self, *, from_node, tx_hex, maxfeerate=0, **kwargs):
         self.scan_tx(from_node.decoderawtransaction(tx_hex))
         return txid
 
+    def create_self_transfer_chain(self, *, chain_length):
```

Comment:
Nice, but needs documentation. You can copy paste the docstring from `make_chain` to here.

Reply:
Done in 17cad448516a6906ff637593ab57df332fade5d2.